lib/internal/errors.js

```diff
@@ -111,6 +111,7 @@ E('ERR_IPC_DISCONNECTED', 'IPC channel is already disconnected');
 E('ERR_IPC_ONE_PIPE', 'Child process can have only one IPC pipe');
 E('ERR_IPC_SYNC_FORK', 'IPC cannot be used with synchronous forks');
 E('ERR_MISSING_ARGS', missingArgs);
+E('ERR_NO_DOMAIN', 'Unable to determine the domain name');
```

Comment:
Although I see it was a copy/paste from the existing error message, so certainly feel free to keep it that way if you'd prefer...